src/types/matching.ts

```diff
@@ -10,3 +10,9 @@ export interface MatchingFeedbackForm {
   message: string;
   isBlockedMatching: boolean;
 }
+
+export interface MatchingInformation {
+  role: 'offer' | 'answer';
+  matchingSocket: string;
+  matchingUser: string;
```

Comment:
ask;
해당 타입이 소켓에서만 사용하는 타입이라면 다음과 같이 수정해도 괜찮지 않을까요?

```suggestion
export interface MatchingSocket {
  role: 'offer' | 'answer';
  socketId: string;
  userId: string;
```

Reply:
수정했습니다!

작업 커밋 : 3431738